code/modules/cargo/packs/misc.dm

```diff
@@ -143,6 +115,136 @@
+					/obj/item/bedsheet/random,
+					/obj/item/bedsheet/random,
+					/obj/item/bedsheet/random,
+					/obj/item/bedsheet/random,
+					/obj/item/bedsheet/random,
+					/obj/item/bedsheet/random) //I'm lazy, and I copy paste stuff.
+	crate_name = "random bedsheet crate"
+
+/datum/supply_pack/misc/coloredsheets
+	name = "Bedsheet Crate (C)"
+	desc = "Give your night life a splash of color with this crate filled with bedsheets! Contains a total of nine different-colored sheets."
+	cost = 1250
+	contains = list(/obj/item/bedsheet/blue,
+					/obj/item/bedsheet/green,
+					/obj/item/bedsheet/orange,
+					/obj/item/bedsheet/purple,
+					/obj/item/bedsheet/red,
+					/obj/item/bedsheet/yellow,
+					/obj/item/bedsheet/brown, 
+					/obj/item/bedsheet/black,
+					/obj/item/bedsheet/rainbow)
+	crate_name = "colored bedsheet crate"
+
+/datum/supply_pack/misc/bicycle
+	name = "Bicycle"
+	desc = "Nanotrasen reminds all employees to never toy with powers outside their control."
+	cost = 1000000
+	contains = list(/obj/vehicle/ridden/bicycle)
+	crate_name = "Bicycle Crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/misc/bigband
+	name = "Big Band Instrument Collection"
+	desc = "Get your sad station movin' and groovin' with this fine collection! Contains nine different instruments!"
+	cost = 5000
+	crate_name = "Big band musical instruments collection"
+	contains = list(/obj/item/instrument/violin,
+					/obj/item/instrument/guitar,
+					/obj/item/instrument/glockenspiel,
+					/obj/item/instrument/accordion,
+					/obj/item/instrument/saxophone,
+					/obj/item/instrument/trombone,
+					/obj/item/instrument/recorder,
+					/obj/item/instrument/harmonica,
+					/obj/structure/piano/unanchored)
+	crate_type = /obj/structure/closet/crate/wooden
+
+/datum/supply_pack/misc/cbtpack
+	name = "CBT Equipment"
+	desc = "(*!&@#CBT is a special term coined by high ranking syndicate operatives for a special form of information extraction. While the training required to use this shipment is highly classified, the distribution of it's contents are not. In addition to this crates unusual contents, we have added a bar of soap.#@*$"
+	hidden = TRUE
+	cost = 2400
+	contains = list(/mob/living/simple_animal/chicken,
+					/obj/item/toy/beach_ball/holoball,
+					/obj/item/melee/baton/cattleprod,
+					/obj/item/soap/syndie)
+	crate_name = "cbt crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/misc/coincrate
+	name = "Coin Crate"
+	desc = "Psssst, hey, you. Yes, you. I've heard that coins can do some special things on your station, give you access to some pretty cool stuff. Here's the deal, you give me some credits, and I give so some coins. Sound like a deal? I'll give you 10 for 10000 creds."
+	contraband = TRUE
+	cost = 10000
+	contains = list(/obj/item/coin/iron)
+	crate_name = "coin crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/misc/coincrate/generate()
```

Comment:
Make them random and lower the cost to 4k~

Reply:
There are coins that can go up to 3000 credits. The crate isn't supposed to be particularly practical or worthwhile it's meant to get coins explicitly for use in a vendor. Making higher value coins possible in the crate just makes it easier for people to get -even more- out of it by shoving a high value coin in a slot machine, getting a refund, and getting dozens upon dozens of extra coins.

Honestly I'm not sure if the slot machine dumps out iron or silver coins, but either way, vendors only take one coin of any value for their extra content. (AFAIK).

Reply:
you can already mass produce coins, 5 coins per 1 sheet of material